src/main/java/com/quirko/app/GameController.java

```diff
@@ -65,7 +84,17 @@ public ViewData onRotateEvent(MoveEvent event) {
 
     @Override
     public void createNewGame() {
+        DifficultySetPanel dfs = new DifficultySetPanel();
+        Enum difficultyLevel_new = dfs.getDifficultyLevel();
+        while(difficultyLevel_new == null){

```

Comment:
Hmm, neden döngüde bekliyoruz?

Reply:
Hocam burada başlangıçta null olarak tanımlı olduğu için difficultyLevel butona basıp değer algılanınca çalışması açısından böyle yazdım. Yukarıda da sleeple bu yüzden saniye bekletiyorum.